cluster/cluster_test.go

```diff
@@ -19,12 +19,25 @@ import (
 	"time"
 
 	"github.com/go-kit/kit/log"
+	"github.com/hashicorp/go-sockaddr"
 	"github.com/stretchr/testify/require"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-func TestJoinLeave(t *testing.T) {
+func TestClusterJoinAndReconnect(t *testing.T) {
+	ip, _ := sockaddr.GetPrivateIP()
+	if ip == "" {
+		t.Skipf("skipping tests because there no private IP address can be found")
```

Comment:
`skipping tests because no private IP address can be found`

Reply:
thanks for catching it!